src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -35,17 +46,52 @@ class VendedorFijo(val ciudadOrigen: Ciudad) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudad == ciudadOrigen
   }
+
+  override fun esInfluyente(): Boolean = false
```

Comment:
No hace falta poner qué tipo devuelve cuando definís el método con `=`:

```suggestion
  override fun esInfluyente() = false
```